slsqp4j/build.gradle

```diff
@@ -25,8 +25,10 @@ task compileC(type: Exec) {
 
     group "Build"
     description "Compiles the C sources."
-    commandLine 'gcc', '-fPIC', '-c', '-o', obj, '-I', System.properties['java.home'] + "/include",
-        '-I', System.properties['java.home'] + "/include/linux", src
+    logger.warn("Java home " + System.properties['java.home'] )
+    commandLine 'gcc-10', '-fPIC', '-c', '-o', obj, '-I', "/Library/Java/JavaVirtualMachines/jdk8/Contents/Home" + "/include",
```

Comment:
Could you wrap this in an if statement that checks the current OS. Can be done via:
```
    OperatingSystem os = org.gradle.nativeplatform.platform.internal.DefaultNativePlatform.currentOperatingSystem;
    if (os.isLinux()) {
        commandLine 'gcc', '-fPIC', '-c', '-o', obj, '-I', System.properties['java.home'] + "/include",
                '-I', System.properties['java.home'] + "/include/linux", src
    } else if (os.isMacOsX()) {
        commandLine 'gcc-10', '-fPIC', '-c', '-o', obj, '-I', "/Library/Java/JavaVirtualMachines/jdk8/Contents/Home" + "/include",
                '-I', "/Library/Java/JavaVirtualMachines/jdk8/Contents/Home" + "/include/darwin", src
    }
```

Reply:
Should be good now.
I replayed the whole installation on BigSur and Mojave OSX version.